lib/dockerfile_parser.rb

```
@@ -0,0 +1,27 @@
+      line = line.strip.split(' ')
+
+      # Remove the FROM keyword
+      line.shift
+
+      # Remove any flags
+      line.reject! { |l| l =~ /^--/ }
+
+      # Remove any comments
+      line.reject! { |l| l =~ /^#/ }
+      {
+        name: line[0].split(':')[0],
```

RuboCop (1.75.5)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)